deploy/kubernetes/operator/Makefile

```diff
@@ -16,12 +16,16 @@
 #
 
 # REGISTRY URL to use all building/pushing image targets
-REGISTRY ?= UNKNOWN_REGISTRY
+REGISTRY ?= ''
+
+ifneq ('', ${REGISTRY})
+REGISTRY := $(addsuffix /, ${REGISTRY})
+endif
 
 MODULES ?= webhook controller
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.22.1
+ENVTEST_K8S_VERSION = 1.24.1
```

Comment:
Maybe you can update it in go.mod as well?

Reply:
Actually my question is why update envtest?

Reply:
It seems that the names of individual fields in k8s v1.24 have been changed, and I recommend a separate PR to change the k8s version.

Reply:
As discussed earlier, why upgrade envtest to 1.24 when you don't need to upgrade k8s to 1.24?

Reply:
> > As discussed earlier, why upgrade envtest to 1.24 when you don't need to upgrade k8s to 1.24? cc @advancedxy
> 
> Yeah, as discussed earlier. Envtest is upgraded to 1.24 to support Apple M1's chips as 1.22 doesn't work for M1 chip.
> 
> The operator itself doesn't need to upgrade to 1.24 and it shouldn't since many K8S clusters are in old versions.


Ok, I get it.